src/index.js

```diff
@@ -1,14 +1,26 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React from "react";
```

Comment:
The import order seems off here.

Reply:
this has been changes following React Redux and then styling ordering